http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/AppComponent.scala

```diff
@@ -156,7 +160,8 @@ object appQuery extends TableQuery(new AppTable(_)) {
         saveApp.app.auditInfo.destroyedDate.getOrElse(dummyDate),
         saveApp.app.auditInfo.dateAccessed,
         namespaceId,
-        diskOpt.map(_.id)
+        diskOpt.map(_.id),
+        saveApp.app.customEnvironmentVariables
```

Comment:
Does this end up getting stored as JSON in the database?

Reply:
I think so because of https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/LeoProfile.scala#L75-L86

Reply:
yeah seems like it. is that okay? i modeled after how we save `customClusterEnvironmentVariables` for runtimes. that code is a bit more complicated so I'm not sure if the env vars also save as json there too. they're stored as a `Map` on the cluster object so i'm assuming so